src/main/kotlin/com/example/probation/validation/annotation/UniqueNewEmail.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.probation.validation.annotation
+
+import com.example.probation.validation.validator.NewEmailValidator
+import javax.validation.Constraint
+import javax.validation.Payload
+import kotlin.reflect.KClass
+
+@MustBeDocumented
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.TYPE, AnnotationTarget.CLASS)
+@Constraint(validatedBy = [NewEmailValidator::class])
+annotation class UniqueNewEmail(
```

Comment:
you do not need two different annotations. it could be just one `UniqueEmail` annotation but with two implementations